BitwardenShared/Core/Auth/Models/Response/ResponseValidationErrorModel.swift

```diff
@@ -12,7 +12,7 @@ struct ResponseValidationErrorModel: Codable, Equatable {
     let error: String
 
     /// A string that provides a description of the error.
-    let errorDescription: String
+    let errorDescription: String?
```

Comment:
:thought_balloon: Was this not showing correctly because the serrver wasn't sending back an `errorDescription` with this specific error?

Reply:
Yeah, correct. The error response from the API seems to only sometimes have this property and it was failing to decode in this scenario.